src/main/java/com/dev/cinema/controllers/MovieController.java

```diff
@@ -0,0 +1,35 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.dto.movie.MovieRequestDto;
+import com.dev.cinema.model.dto.movie.MovieResponseDto;
+import com.dev.cinema.service.MovieService;
+import com.dev.cinema.service.mapper.MovieMapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MovieController {
+    private final MovieService movieService;
+    private final MovieMapper movieMapper;
+
+    public MovieController(MovieService movieService, MovieMapper movieMapper) {
+        this.movieService = movieService;
+        this.movieMapper = movieMapper;
+    }
+
+    @GetMapping("/movies")
+    public List<MovieResponseDto> getAllMovies() {
+        return movieService.getAll().stream()
+                .map(movieMapper::toMovieDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping("/add-movie")
```

Comment:
guess POST has the same mapping as get: Add a movie - POST: /movies (from HW)

Reply:
it's working fine (I tested with postman). But naming convention... thanks, just changed